pkg/cmd/pulumi/config_env_init.go

```diff
@@ -0,0 +1,333 @@
+// Copyright 2016-2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"text/template"
+
+	"github.com/charmbracelet/glamour"
```

Comment:
Little sad we've got some commands using bubbletea now and now ESC commands using glamour. We're going to need a display sanitisation pass at some point.

Reply:
Yeah, it'd be nice to take better advantage of `bubbletea` for some of our more interactive bits. In this case I don't think there'd be anything to change--we'd still use `glamour`, as it's specifically oriented around rendering Markdown.

Reply:
OK, if it's additive rather than a replacement makes a lot more sense to use both. We can do bubble tea work iteratively command by command as we want.